pkg/cluster/prepare.go

```diff
@@ -109,7 +110,11 @@ func (p *Preparer) PreFlightCheck() error {
 	args := []string{"get", "namespace", p.FluxNamespace, "--output", "name"}
 	if output, err := p.Runner.Run(kubectlCmd, args...); err != nil {
 		fmt.Println("\nOutput from kubectl command: ", string(output))
-		return fmt.Errorf("failed to get flux namespace: %w", err)
+		if p.IgnorePreflightErrors {
+			fmt.Println("WARNING: failed to get flux namespace. Flux is required for profiles to work.")
+		} else {
+			return fmt.Errorf("failed to get flux namespace: %w", err)
```

Comment:
shall` we print a hint about using this flag? e.g. 
```
failed to get flux namespace: <error>
if you wish to proceed with the installation without flux/pre-flight-checks please see the  --ignore-preflight-checks
```

Reply:
Yeah that makes sense

Reply:
Added.